README.md

```diff
@@ -98,28 +98,37 @@ See [SECURITY.md](SECURITY.md) for more details.
-|     [![emonddr]](https://github.com/emonddr)     |         [![agnes512]](https://github.com/agnes512)         | [![deepakrkris]](https://github.com/deepakrkris) |
-|                  Denny Bartelt                   |                    Douglas McConnachie                     |                  Rifa Achrinza                   |
-|     [![derdeka]](https://github.com/derdeka)     |         [![dougal83]](https://github.com/dougal83)         |    [![achrinza]](https://github.com/achrinza)    |
-|                 Francisco Buceta                 |                       Matthew Schnee                       |                                                  |
-|    [![frbuceta]](https://github.com/frbuceta)    |          [![mschnee]](https://github.com/mschnee)          |                                                  |
+|                                            |                                                            |                                                  |
+| :----------------------------------------: | :--------------------------------------------------------: | :----------------------------------------------: |
+|                 Diana Lau                  |                         Janny Hou                          |                    Hage Yaapa                    |
+|   [![dhmlau]](http://github.com/dhmlau)    |         [![jannyhou]](http://github.com/jannyHou)          | [![hacksparrow]](https://github.com/hacksparrow) |
+|                 Agnes Lin                  |                       Mario Estrada                        |                   Hugo Da Roit                   |
+| [![agnes512]](https://github.com/agnes512) | [![marioestradarosa]](https://github.com/marioestradarosa) |        [![yaty]](https://github.com/yaty)        |
+|              Dominique Emond               |                       Denny Bartelt                        |               Douglas McConnachie                |
+|  [![emonddr]](https://github.com/emonddr)  |          [![derdeka]](https://github.com/derdeka)          |    [![dougal83]](https://github.com/dougal83)    |
+|               Rifa Achrinza                |                      Francisco Buceta                      |                  Matthew Schnee                  |
+| [![achrinza]](https://github.com/achrinza) |         [![frbuceta]](https://github.com/frbuceta)         |     [![mschnee]](https://github.com/mschnee)     |  |
 
 See
 [all contributors](https://github.com/strongloop/loopback-next/graphs/contributors).
 
+### Alumni
+
+- [@ritch](http://github.com/ritch)
+- [@superkhau](https://github.com/superkhau)
+- [@rashmihunt](https://github.com/rashmihunt)
+- [@kjdelisle](https://github.com/kjdelisle)
+- [@virkt25](https://github.com/virkt25)
+- [@shimks](https://github.com/shimks)
+- [@nabdelgadir](https://github.com/nabdelgadir)
+- [@deepakrkris](https://github.com/deepakrkris)
+
```

Comment:
- [Biniam](https://github.com/b-admike)

:D

Reply:
Do you want to fix the PR? I assume @bajtos is on vacation for some time.

Reply:
Fixed!